src/config/theme.ts

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+import 'dotenv/config';
+
+export const themeConfig = {
+  colors: {
+    primary: process.env.NEXT_PUBLIC_COLOR_PRIMARY,
```

Comment:
Does it not make sense to also move font config in this file and then use this config in the code? instead of directly using the NEXT_PUBLIC_SELECTED_FONT_LIGHT or NEXT_PUBLIC_SELECTED_FONT_MEDIUM in the code

Reply:
@inderps I moved the fonts to theme file too.